### PR TITLE
Preset Boxes File De-concatenation

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

image

Hey there,

We had one file that was like eighteen-hundred (1800) lines full of just... box presets. There was no rhyme or reason to where anything was in the list, it just sorta got to the point where new features were found near the bottom with zero grouping. So, let's de-concatenate this massive file and give it some proper grouping.

While I was in the area, I did some file cleanup, using `snake_case` instead of whatever the fuck some vars were, alphabetizing and cleaning up lists to have trailing commas, that sorta stuff. Let me know if I broke something somewhere.

## Why It's Good For The Game

Maintainability to have stuff well-organized. Code improvement to have all the minor codey stuff fixed up while in the area. Some documentation where I felt like some was missing. Alphabetization is good to have. Lists shouldn't look ugly like they were, they should be clean :).

## Changelog

Nothing player facing (until it comes out that something I did broke something somehow).

### CODE REVIEW

File: code/game/objects/items/storage/boxes/job_boxes.dm [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+	..()
+	user.changeNext_move(CLICK_CD_MELEE)
+	playsound(loc, SFX_RUSTLE, 50, TRUE, -5)
+	user.visible_message(span_notice("[user] hugs \the [src]."),span_notice("You hug \the [src]."))
+
+/obj/item/storage/box/hug/black
+	icon_state = "hugbox_black"
+	illustration = "heart_black"
+
+// clown box & honkbot assembly
```
Comment (Time-Green) [inline code/game/objects/items/storage/boxes/job_boxes.dm:133]: Doc comment these while you’re at it?

File: code/game/objects/items/storage/boxes/security_boxes.dm [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+	desc = "Contains an assortment of fireworks."
+	illustration = "sparkler"
+
+/obj/item/storage/box/fireworks/PopulateContents()
+	for(var/i in 1 to 3)
+		new/obj/item/sparkler(src)
+		new/obj/item/grenade/firecracker(src)
+	new /obj/item/toy/snappop(src)
+
+/obj/item/storage/box/fireworks/dangerous
```
Comment (Time-Green) [inline code/game/objects/items/storage/boxes/security_boxes.dm:269]: You don’t actually need this, just having the proc creates the subtype

**Comment by san7890** [comment]
WAIT WHAT THE FUCK I HIT "DELETE" COMMENT IN VSC AND IT DELETED IT ON THE PR? THAT'S SOME STUPID FUCKING BULLSHIT

File: code/game/objects/items/storage/boxes/security_boxes.dm (hunk 2 above)
Comment (san7890) [reply code/game/objects/items/storage/boxes/security_boxes.dm:269]: i have a better idea for this actually

**Comment by Time-Green** [comment]
> WAIT WHAT THE FUCK I HIT "DELETE" COMMENT IN VSC AND IT DELETED IT ON THE PR? THAT'S SOME STUPID FUCKING BULLSHIT
> 
> Completely unintentional, I thought it would merely delete the comment on my local rather than fucking own the PR-level comment. Apologies.

Bro what you can see PR comments in VSC??? 

It’s fine, nothing a bit of therapy can’t solve

**Comment by Time-Green** [comment]
> WAIT WHAT THE FUCK I HIT "DELETE" COMMENT IN VSC AND IT DELETED IT ON THE PR? THAT'S SOME STUPID FUCKING BULLSHIT
> 
> Completely unintentional, I thought it would merely delete the comment on my local rather than fucking own the PR-level comment. Apologies.

Bro what you can see PR comments in VSC??? 

It’s fine, nothing a few years of therapy won’t solve

**Pull request merged**